index.js

```diff
@@ -405,6 +405,12 @@ class LiveStrategyExecution extends EventEmitter {
     this.perfManager.on('update', () => this._emitStrategyExecutionResults('perf', { price: this.priceFeed.price.toNumber() }))
   }
 
+  async restartExecution (ws2Manager) {
+    this.ws2Manager = ws2Manager
```

Comment:
Do you think we should close the previous resources before starting the new ones? Listeners like the ones in `_subscribeCandleAndTradeEvents` could become memory leaks

Reply:
Disconnecting previous ws in strategy manager.